### PR TITLE
hook up evaluator to main

### PR DESCRIPTION
**Pull request by philipcraig**

### CODE REVIEW

File: src/io.rs [hunk 1]
```diff
@@ -65,10 +63,12 @@ fn process_text<W: Write>(text: &str, mut out: W) -> Result<()> {
         )?;
     }
     let parser = Parser::new(tokens);
-    for err in parser.ast().errors() {
+    let ast = parser.ast();
+    for err in ast.errors() {
```
Comment (jbcoe) [inline src/io.rs:67]: Do we want to quit if there are AST errors? Evaluating a broken AST seems wrong.

File: src/io.rs [hunk 2]
```diff
@@ -8,27 +9,24 @@ use std::{
     io::{self, Read, Write},
 };
 
-/// process each filepath in the input Vec, or stdin if empty
-pub fn process_files(filepaths: Vec<String>) -> Result<()> {
+/// process first filepath in the input Vec, or stdin if empty
```
Comment (jbcoe) [inline src/io.rs:12]: Should we fail if there are multiple files?

**Review by jbcoe — APPROVED** [review]

File: src/io.rs (hunk 1 above)
Comment (philipcraig) [reply src/io.rs:67]: yes, coming in a future PR

File: src/io.rs (hunk 2 above)
Comment (philipcraig) [reply src/io.rs:12]: yes, coming in a future PR that hooks up `structopt`

**Pull request merged**